app/jobs/create_onboarding_notifications_job.rb

```diff
@@ -0,0 +1,53 @@
+class CreateOnboardingNotificationsJob < ApplicationJob
+  queue_as :background
+
+  OnboardingEmail = Struct.new(:day, :slug, keyword_init: true) do
+    def notification_type
+      "onboarding_#{slug}".to_sym
+    end
+  end
+
+  # 0 would be immediately after signing up
+  # 1 is a day after signing up, etc
+  #
+  # Each slug should be paired with a User::Notifications::Onboarding{$SLUG}Notification
+  EMAILS = {
+    1 => :community,
+    3 => :fundraising
```

Comment:
Even though it is failing right now, it is looking for the right thing, so that's good!

```shell
ntize': uninitialized constant User::Notifications::OnboardingFundraisingNotification (NameError)

      Object.const_get(camel_cased_word)
            ^^^^^^^^^^
Did you mean?  User::Notifications::OnboardingCommunityNotification
```